lessons/Projects/project-snack-habits/langs/en-us/index.adoc

```diff
@@ -0,0 +1,194 @@
+- What do you Notice? What do you Wonder?
+- With your partner complete the third column of the table by deciding what data type each variable will be.
+}
+
+Confirm that students have correctly identified the data types.
+
+@teacher{
+Make sure you've made a copy of @link{https://docs.google.com/forms/d/1PklP5FhpsWlUrQU6UaV0OKYxzq_Pz5YheLoiGs3yTeM/copy, this Google form} and shared a link with your students.
+}
+
+@lesson-instruction{ 
+- Your task is to complete the google form for every snack you eat over the next 5 days. 
+- Please save the link I shared with you somewhere that will be easy for you to find. (In "Notes" on your phone? In a text message? In an email?)
+- If you eat a snack when you don't have time to fill out the form, please take a picture of it and/or make a note to help you fill out the form later on.
+- We will be looking at the spreadsheet together in our next class to see how our dataset is shaping up.
+}
+
+@teacher{You may want to have students complete @handout{snack-habits-check-in.adoc} part way through the data collection process.}
+
+=== Investigate
+
+@teacher{
+To access your class' data:
+- *Link the Google Form Responses to a Google Sheet* from the Responses of the form by clicking the button that reads *Link to Sheet*. The sheet will automatically continue to populate the Google Sheet each time the form gets submitted.
+- When your class is done collecting data, 
+  * duplicate the sheet in a new tab 
+  * name the new tab "cleaned" 
+  * remove the timestamp column and the image column
+  * make sure the sharing settings are set to "Anyone with the link can view"
+@ifproglang{pyret}{
+  * copy the url into the @starter-file{snack} to import the data, publish the file and share a link with your students
+}
+@ifproglang{codap}{
+  * import the spreadsheet into CODAP and share a link with your students
+}
+
+We have set the form up with data verification for most questions to minimize the amount of cleaning you will have to do, but we would expect that the one word answers to "Why are you eating this snack?" will require some attention to address: 
+
+- typos
+- inconsistent capitalization
+- words that mean the same thing as each other and should be combined 
+- perhaps there are a number of nonsense words that might make more sense to be replaced by "idk" why I'm snacking?
+
+If you have the time, we encourage you to project the spreadsheet and tackle cleaning the data with your students, asking:
+
+- What inconsistencies do you see in the data?
+- How should we address them?
+
+Ideally, by the time you're done you could make a pie-chart of the "why" column and the breakdown of reasons would be informative. 
+}
+
+@lesson-instruction{
+- Now that we have finished collecting our data, let's see what we can learn about our habits!
+- Open the starter file I shared with you and use it to complete @handout{data-cycle-categorical.adoc}
+}
```

Comment:
add a note that students should save displays / releveant code to add to their final proejct. (I would even include a link to the slide deck for teachers - like me! - who would want to have their students fill in the slide deck as they work.)